libs/wingc/src/type_check.rs

```diff
@@ -1304,6 +1305,17 @@ impl<'a> TypeChecker<'a> {
 				// Create constructor environment and prime it with args
 				let mut constructor_env = SymbolEnv::new(Some(env), constructor_sig.return_type, false, env_flight, stmt.idx);
 				self.add_arguments_to_env(&constructor.parameters, constructor_sig, &mut constructor_env);
+				// Prime the constructor environment with `this`
```

Comment:
Is `this` also supported within methods?

Reply:
That's already working.